website/blog/2023-04-26-deprecating-dbt-metrics.md

```diff
@@ -0,0 +1,48 @@
+description: "We are bidding adieu to dbt_metrics and moving forward with MetricFlow! Discover how this new source-available project lays the foundation for the dbt Semantic Layer. Let's dive in!"
+slug: deprecating-dbt-metrics
+
+authors: [callum_mccann]
+
+tags: [dbt product updates]
+hide_table_of_contents: false
+
+date: 2023-04-26
+is_featured: true
+---
+
+## Deprecating dbt_metrics
+
+Hello, my dear data people.
+
+If you haven’t read Nick’s blog post about what’s coming in the future of the dbt Semantic Layer, I highly recommend you read through that as it gives helpful context around what the future holds. LINK TO COME.
+
+With that said, it has come time for us to bid adieu to our beloved dbt_metrics package. **Upon the release of dbt-core v1.6 in late July, we will be deprecating support for the dbt_metrics package.** 
+
+With the upcoming integration with MetricFlow, we're on the cusp of a new era for the dbt Semantic Layer. And if we’re honest with ourselves, it is a brighter future than the dbt_metrics package would have been able to support.
+
+We know that some of you have been using the package to help serve your organizations. **We will be providing migration scripts to help reduce the complexity of upgrading to the new specs.** 
+
+If you’re interested in getting ahead of the transition, we recommend that you start familiarizing yourself with the new spec in this [Github Discussion](https://github.com/dbt-labs/dbt-core/discussions/7456). If you’re even more curious, you can read the MetricFlow documentation, although some elements will change in the coming months as we work on the integration. I’m confident that, with a little time, you'll come to love MetricFlow far more than you did dbt_metrics.
+
+Before we get into the technical details around why we’re deprecating the package, I want to give a big thank you to everyone from the community who used or contributed to the dbt_metrics package over the last year. From the bottom of my heart, I’ve loved hearing from all of you and discussing ways to help you solve your organizations data problems. It’s been a dream come true to work in this area and it wouldn’t have been possible without all of you lovely folks.
+
+**Why We’re Deprecating**
+With all that said, let’s dig into the exact reasons why we’re deprecating the package.
+
+- **Duplicative Functionality:** MetricFlow and dbt_metrics share a common purpose – a simplified and standard way to generate SQL to query metrics. However, MetricFlow does this much more effectively, with advanced features and a more robust foundation.
+- **The Jinja Trap:** Relying on Jinja macros for generating SQL in dbt_metrics proved to be sub-optimal and restrictive. This approach limited the package's potential and made it difficult to scale and customize for each new metric type and piece of functionality that we wanted to add. In contrast, MetricFlow's Python implementation offers a far more flexible and expressive solution.
```

Comment:
```suggestion
- **The Jinja trap:** Relying on Jinja macros for generating SQL in dbt_metrics proved to be sub-optimal and restrictive. This approach limited the package's potential and made it difficult to scale and customize for each new metric type and piece of functionality that we wanted to add. In contrast, MetricFlow's Python implementation offers a far more flexible and expressive solution.
```

Reply:
yeah I would adjust all bullets for this capitalization standard, good callout @mirnawong1 !